### PR TITLE
Creating ActiveBenefitDetails component

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Creating ActiveBenefitDetails component that works for all active benefits
- Adding CPP active Card

### CODE REVIEW

File: components/molecules/PendingBenefitDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+
+const PendingBenefitCard = (props) => {
+  const t = props.locale === 'en' ? en : fr
+
+  return (
+    <div className="grid grid-cols-1 divide-y divide-grey-500 font-display my-4">
+      <section>
+        <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-7 sm:gap-4 sm:mx-auto sm:mt-4">
```
Comment (arsanig) [inline components/molecules/PendingBenefitDetails.jsx:13]: There's both flex and grid assigned on this and I don't think you need the flex

File: components/molecules/PendingBenefitDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          <div
+            id="claimUpdates"
+            className="col-span-2 sm:col-span-3 lg:col-span-2 sm:w-60 ml-7 mb-6 sm:mb-3"
```
Comment (arsanig) [inline components/molecules/PendingBenefitDetails.jsx:16]: I think this broke the mobile work that was completed as part of PR #181 

image

File: components/molecules/PendingBenefitDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <VerticalStepper
+              benefitStatus={props.benefit.status}
+              lastUpdates={props.benefit.lastUpdates}
+              locale={props.locale}
+            />
+          </div>
+
+          <div
+            id="details"
+            className="col-span-5 sm:col-span-4 lg:col-span-5 sm:grid mr-5"
```
Comment (arsanig) [inline components/molecules/PendingBenefitDetails.jsx:27]: `className="col-span-5 sm:col-span-4 lg:col-span-5 sm:grid mr-5"`

Don't need col-span-5, twice here. Something like below will accomplish the same thing:
`className="col-span-4 mr-5 sm:grid sm:col-span-5"`

File: components/molecules/PendingBenefitDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+          >
+            <div id="application_details" className="ml-5 mb-5 sm:mb-0">
+              <p className="font-bold text-xl">{t.applicationDetails}</p>
+              <p className="font-normal text-base">
+                {props.benefit.statusDescription}
+              </p>
+            </div>
+            <div id="application_description" className="ml-5 mb-5 sm:mb-0">
```
Comment (arsanig) [inline components/molecules/PendingBenefitDetails.jsx:35]: Instead of `ml-5` per item here, just put it on the parent `<div>`

**Review by arsanig — CHANGES_REQUESTED** [review]
Looks good! 👍🏻 Though I think there were a few things missed with the local merged you mentioned you performed, see feedback below.

File: components/molecules/PendingBenefitDetails.jsx (hunk 1 above)
Comment (amirpourpanahi) [reply components/molecules/PendingBenefitDetails.jsx:13]: flex is removed

File: components/molecules/PendingBenefitDetails.jsx (hunk 2 above)
Comment (amirpourpanahi) [reply components/molecules/PendingBenefitDetails.jsx:16]: Do you mean the Left Margin?
I just changed the name of this file. I worked on Active card.

File: components/molecules/PendingBenefitDetails.jsx (hunk 3 above)
Comment (amirpourpanahi) [reply components/molecules/PendingBenefitDetails.jsx:27]: mobile-screens
https://tailwindcss.com/docs/responsive-design

File: components/molecules/PendingBenefitDetails.jsx (hunk 4 above)
Comment (amirpourpanahi) [reply components/molecules/PendingBenefitDetails.jsx:35]: Done

File: components/molecules/PendingBenefitDetails.jsx (hunk 2 above)
Comment (arsanig) [reply components/molecules/PendingBenefitDetails.jsx:16]: This file is different than its previous `PendingBenefitCard.jsx` something changed somewhere.

**Review by nabil386 — APPROVED** [review]

File: components/molecules/PendingBenefitDetails.jsx (hunk 3 above)
Comment (arsanig) [reply components/molecules/PendingBenefitDetails.jsx]: Ok sounds good

**Pull request closed without merging**